### PR TITLE
update strangepg and its requirements

### PR DESCRIPTION
**Pull request by qwx9**
Update to latest and fix requirements.

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

### General instructions

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

### Instructions for avoiding API, ABI, and CLI breakage issues
Conda is able to record and lock (a.k.a. pin) dependency versions used at build time of other recipes.
This way, one can avoid that expectations of a downstream recipe with regards to API, ABI, or CLI are violated by later changes in the recipe.
If not already present in the meta.yaml, make sure to specify `run_exports` (see [here](https://bioconda.github.io/contributor/linting.html#missing-run-exports) for the rationale and comprehensive explanation).
Add a `run_exports` section like this:

```yaml
build:
  run_exports:
    - ...

```

with `...` being one of:

| Case                             | run_exports statement                                               |
| -------------------------------- | ------------------------------------------------------------------- |
| semantic versioning              | `{{ pin_subpackage("myrecipe", max_pin="x") }}`     |
| semantic versioning (0.x.x)      | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}`   |
| known breakage in minor versions | `{{ pin_subpackage("myrecipe", max_pin="x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| known breakage in patch versions | `{{ pin_subpackage("myrecipe", max_pin="x.x.x") }}` (in such a case, please add a note that shortly mentions your evidence for that) |
| calendar versioning              | `{{ pin_subpackage("myrecipe", max_pin=None) }}`    |

while replacing `"myrecipe"` with either `name` if a `name|lower` variable is defined in your recipe or with the lowercase name of the package in quotes.

### Bot commands for PR management

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by qwx9** [comment]
@BiocondaBot please add label

**Comment by bgruening** [comment]
The test are not very powerful. Do you mind fetching the binaries and testing them. I'm especially concerned about removing the EU exports.

**Comment by bgruening** [comment]
@BiocondaBot please fetch artifacts

**Comment by bgruening** [comment]
@BiocondaBot please fetch artifacts

**Comment by qwx9** [comment]
@bgruening I agree about the tests, in fact conda breakages have been related to conflicts with system X11 or GL libraries on initialization. However, it's a graphical-only application, would you have any suggestions for better tests?

**Comment by bgruening** [comment]
Does it not have any --help or --version parame? You could start the application and expect an defined error. Maybe that will work?

**Comment by qwx9** [comment]
Hm, it only creates a context once it's done reading an input file, but that might work. I'll amend once I make this work. Thanks!

**Comment by qwx9** [comment]
@bgruening I added more sensible tests to check if the tools run, but I don't see any good way of checking if a GL context can successfully be created. Other recipes don't seem to do more.

File: recipes/strangepg/meta.yaml [hunk 1]
```diff
@@ -47,9 +47,8 @@ requirements:
     - xorg-libxrandr
 
 test:
-  commands:
```
Comment (bgruening) [inline recipes/strangepg/meta.yaml:50]: you can leave those in if you like.

**Comment by bgruening** [comment]
Looks better, thanks! Did you get a change to the package https://github.com/bioconda/bioconda-recipes/pull/50552#issuecomment-2336617876

**Comment by qwx9** [comment]
Sorry, do you mean if I tested the binaries in those artifacts? If so, yes, they worked as expected.

**Comment by bgruening** [comment]
Yes, that's what I meant. Thanks!

**Pull request merged**